mne/decoding/transformer.py

```diff
@@ -248,6 +248,12 @@ def inverse_transform(self, X, y=None):
 class PSDEstimator(TransformerMixin):
     """Compute power spectrum density (PSD) using a multi-taper method
 
+    This structures data so that it can be easily incorporated into
+    scikit-learn pipelines. It relies heavily on the multitaper_psd
```

Comment:
Yeah, that's also why I didn't put it in there in the first place.
Functions like spectral_connectivity (and the newly-modified
compute_epochs_psd) handle it by having a bunch of kwargs for the
respective functions (e.g., mt_adaptive, mt_normalize). I decided to pass
on that this time around though it shouldn't be a big deal to extend the
function to make it work with a method parameter.

On Sun, Dec 20, 2015 at 3:55 AM, Mainak Jas <notifications@github.com>
wrote:

> In mne/decoding/transformer.py
> <https://github.com/mne-tools/mne-python/pull/2710#discussion_r48102657>:
>
> > @@ -248,6 +248,12 @@ def inverse_transform(self, X, y=None):
> >  class PSDEstimator(TransformerMixin):
> >      """Compute power spectrum density (PSD) using a multi-taper method
> >
> > +    This structures data so that it can be easily incorporated into
> > +    scikit-learn pipelines. It relies heavily on the multitaper_psd
>
> hmm ... actually, thinking about it some more -- the problem with a method
> argument is that you will have to pass the different options for
> multitaper / welch too. Maybe, we can leave it for now ...
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mne-tools/mne-python/pull/2710/files#r48102657>.
>